lib/rules/require-spaces-in-anonymous-function-expression.js

```diff
@@ -117,7 +153,7 @@ module.exports.prototype = {
                     message: 'Missing space before opening curly brace'
                 });
             }
-        });
+        }.bind(this));
```

Comment:
Nit-pick: we usually don't use bindings, that is why we have those lines:
```js
var beforeOpeningRoundBrace = this._beforeOpeningRoundBrace;
var beforeOpeningCurlyBrace = this._beforeOpeningCurlyBrace;
```
Would you mind using `this.exceptions` property in the similar manner?